chapters/functions.tex

```diff
@@ -1341,17 +1336,17 @@ \subsection{Using the Derivative Annotation}\doublelabel{using-the-derivative-an
   ...
 \end{lstlisting}
 the equation
-
-(...,y(t),...)=foo0(...,x(t),b,...);
-
+\begin{lstlisting}[language=modelica]
+(..., y(t), ...) = foo0(..., x(t), b,...);
+\end{lstlisting}
 implies that:
-
-(...,d y(t)/dt,...)=foo1(...,x(t),b,..., ...,d x(t)/dt,...);
-
-(...,d\^{}2 y(t)/dt\^{}2,...)=foo2(...,x(t),b,...,d x(t)/dt,...,
-...,d\^{}2 x(t)/dt\^{}2,...);
-
-{]}
+\begin{lstlisting}[language=modelica]
```

Comment:
Same here: I wouldn't view this as Modelica-code, due to y(t). Thus I would propose to keep as before for now.

Reply:
Same fix as above.